tests/integration/frameworks/test_frameworks.py

```diff
@@ -139,7 +144,12 @@ def test_load(
                     key: inp.preprocess(kwarg)
                     for key, kwarg in inp.input_kwargs.items()
                 }
-                out = getattr(model, method)(*args, **kwargs)
+                if test_model.model_method_caller:
+                    out = test_model.model_method_caller(
+                        test_model, method, args, kwargs
+                    )
+                else:
+                    out = getattr(model, method)(*args, **kwargs)
```

Comment:
cc @sauyon some changes for `FrameworkTest` for onnx or other more flexible frameworks.